doc/services/crypto/psa_crypto.rst

```diff
@@ -0,0 +1,195 @@
+
+  The API provides all primitives related to storage encryption, block
+  or file-based, with master encryption keys stored inside a key store.
+
+* Network Credentials
+
+  The API provides network credential management inside a key store,
+  for example, for X.509-based authentication or pre-shared keys on
+  enterprise networks.
+
+* Device Pairing
+
+  The API provides support for key agreement protocols that are often
+  used for secure pairing of devices over wireless channels. For
+  example, the pairing of an NFC token or a Bluetooth device might use
+  key agreement protocols upon first use.
+
+* Secure Boot
+
+  The API provides primitives for use during firmware integrity and
+  authenticity validation, during a secure or trusted boot process.
+
+* Attestation
+
+  The API provides primitives used in attestation
+  activities. Attestation is the ability for a device to sign an array
+  of bytes with a device private key and return the result to the
+  caller. There are several use cases; ranging from attestation of the
+  device state, to the ability to generate a key pair and prove that it
+  has been generated inside a secure key store. The API provides access
+  to the algorithms commonly used for attestation.
+
+* Factory Provisioning
+
+  Most IoT devices receive a unique identity during the factory
+  provisioning process, or once they have been deployed to the
+  field. This API provides the APIs necessary for populating a device
+  with keys that represent that identity.
+
+Usage considerations
+********************
+
+
+* Always check for errors
+
+  Most functions in the Crypto API can return errors. All functions
+  that can fail have the return type psa_status_t. A few functions
+  cannot fail, and thus, return void or some other type.
+
+  If an error occurs, unless otherwise specified, the content of the
+  output parameters is undefined and must not be used.
+
+  Some common causes of errors include:
+
+  * In implementations where the keys are stored and processed in a
+    separate environment from the application, all functions that need
+    to access the cryptography processing environment might fail due
+    to an error in the communication between the two environments.
+
+  * If an algorithm is implemented with a hardware accelerator, which
+    is logically separate from the application processor, the
+    accelerator might fail, even when the application processor keeps
+    running normally.
+
+  * Most functions might fail due to a lack of resources. However,
+    some implementations guarantee that certain functions always have
+    sufficient memory.
+
+  * All functions that access persistent keys might fail due to a
+    storage failure.
+
+  * All functions that require randomness might fail due to a lack of
+    entropy. Implementations are encouraged to seed the random
+    generator with sufficient entropy during the execution of
+    psa_crypto_init(). However, some security standards require
```

Comment:
```suggestion
    `psa_crypto_init()`. However, some security standards require
```

Reply:
@valeriosetti it's actually the below :) But nice catch anyway!

```suggestion
    ``psa_crypto_init()``. However, some security standards require
```